src/components/JoinUs/OpenPositions.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import remcalc from 'remcalc'
+import { Row, CompensatedCol, CompensatedRow } from '../grid'
+import styled from 'styled-components'
+import JobsByLocation from '../jobsByLocation'
+import { Padding } from 'styled-components-spacing'
+import { Paragraph, H2, H5 } from '../Typography'
+import { FirstColumn, SecondColumn, Section } from './elements'
+import ExternalAnchor from '../Common/ExternalAnchor'
+import Hr from '../Common/Hr'
+import StyledLink from '../styledLink'
+
+const Line = styled(Hr)`
+  width: ${remcalc(60)};
+`
+
+const JobLinkWrapper = styled.div`
```

Comment:
We have `styled-flex-component` that we can use instead of a `div` + `display: flex`.

Again, not because either is necessarily better or worse. Just for consistency across our codebase.

Reply:
👍